### ISSUE TITLE
The color contrast ratio of the focus indicator on "Back" button with respect to background is less than 3:1

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#1856]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**-A11yMAS;-A11yTCS;-VisualStudioCode-Win32-Apr2021;-Visual Studio Code Client;-A11yWCAG2.1;-A11ySev2;-WCAG1.4.11;-DesktopApp;-Win10;-ColorContrast;

--- Environment Details:
Visual Studio Code
Operating System: Windows 10 Pro 21H1

--- Steps to Reproduce:

Launch the application.

1. Download the  quickinput-sample extension here: https://github-com/microsoft/vscode-extension-samples/tree/main/quickinput-sample
2. F5 the sample extension.
3. Run "Quick Input Samples" from the command palette
4. Pick "multiStepInput" and then TAB to any option and select it using ENTER key.
5. The next step will have a "Back" button that demonstrates the original issue.
6. TAB to "Back" button.
7. Open AI4D color contrast analyzer and check the color contrast ratio of the focus indicator on "Back" button with respect to background

--- Actual:

The color contrast ratio of the focus indicator on "Back" button with respect to background is 2.663:1 which is less than 3:1.

--- Expected:
The color contrast ratio of the focus indicator on "Back" button with respect to background should be greater than or equal to 3:1.

--- User Impact:
The low vision users won't be able to see the focus on the "Back" button properly.
